payu/fsops.py

```diff
@@ -27,6 +27,11 @@
 # Delete this once this bug in Lustre is fixed
 CHECK_LUSTRE_PATH_LEN = True
 
+# File extensions to script interpreters
+EXTENSION_TO_INTERPRETER = {'.py': sys.executable,
```

Comment:
Does line 31 need to detect the Python `sys.executable` or potentially a _virtual environ_ executable?

Reply:
I think it detects the python that is being used in the environment used to run payu.

Reply:
For future reference:
* From the system Python, `sys.executable` resolves to something `/usr/bin/python3`
* Inside a virtualenv, `sys.executable` resolves to the virtualenv `bin` dir & the symlink to the python binary